dictionary/templates/dictionary/search.html

```diff
@@ -0,0 +1,28 @@
+{% extends "layout.html" %}
+{% load bootstrap3 %}
+
+{% block title %}Search{% endblock title %}
+
+
+{% block body %}
+    <form method="POST" action="" >
+        {% csrf_token %}
+        {#  form.as_p #}
+        {% bootstrap_form form %}
+        <input type="submit" value="search" class="btn btn-primary">
```

Comment:
Another option here which I think is a little cleaner for the submit button is to do this:

`<button type="submit">search</button>`